libfaketime/__init__.py

```diff
@@ -208,7 +204,13 @@ def __exit__(self, *exc):
                 os.environ['FAKETIME'] = self._prev_spec
             else:
                 del os.environ['FAKETIME']
-                end_callback(self)
```

Comment:
huh, looks like this was a bug before, yeah? it should have been one level out?

Reply:
Yes I suppose, else that would imply that the end callback is only called if there was not a previous `FAKETIME` environment var defined, and I am not sure how it makes sense.

Reply:
I can do a fix for this in a separate PR if you prefer.

Reply:
looks like it's been fixed. I just wanted to double check it.